sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -465,3 +540,4 @@ The behavior of the SYCL buffer destructor depends on the Ownership flag. As wit
 |8|2022-01-06|Artur Gainullin|Introduced make_buffer() API
 |9|2022-05-12|Steffen Larsen|Added device member to queue input type
 |10|2022-08-18|Sergey Maslov|Moved free_memory device info query to be sycl_ext_intel_device_info extension
+|11|2023-02-05|Rajiv Deodhar|Added make_queue2 and get_native2 methods for SYCL queues that use immediate commandlists

```

Comment:
are make_queue2 good names? or would it better to change the "2" to something more descriptive?

Reply:
The names are temporary until the next ABI change, when once again there will only be one make_queue and the new functionality will be folded into it. For that reason I have not tried to be descriptive or creative about the name.

Reply:
Maybe ,
`make_queue_immediate` `get_native_immediate` ? something more self-descriptive still.

Reply:
The new make_queue can create a queue with _either_ standard or immediate command lists. Similarly, the new get_native can return _either_ an L0 command queue handle or an L0 command list handle.
Perhaps `make_queue_standard_or_immediate, get_native_standard_or_immediate`.

Reply:
Is this a Level Zero specific query? `make_level_zero_queue`?

Reply:
Since the new APIs instantiate templates with the "backend::ext_oneapi_level_zero" specialization it is already quite clear that these are for Level Zero.
The new make_queue allows creating a SYCL queue that will use L0 immediate command lists or the normal ones, so I am proposing a name that makes that explicit.